gen/myq_printsvc

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/perl                                                                                                                     
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Text::CSV_XS;
+
+local $::SERVICE_NAME = "myq_printsvc";
+local $::PROTOCOL_VERSION = "3.1.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+# Output file name
+my $output_fileName = "$DIRECTORY/import_users-perun".".csv";
+
+#Constants
+our $A_FIRST_NAME;  *A_FIRST_NAME = \'urn:perun:user:attribute-def:core:firstName';
+our $A_LAST_NAME;  *A_LAST_NAME = \'urn:perun:user:attribute-def:core:lastName';
+our $A_MAIL;  *A_MAIL = \'urn:perun:user:attribute-def:def:preferredMail';
+our $A_LOGIN; *A_LOGIN = \'urn:perun:user_facility:attribute-def:virt:login';
+our $A_CHIPNUMBERS; *A_CHIPNUMBERS = \'urn:perun:user:attribute-def:def:chipNumbers';
+our $A_R_NAME; *A_R_NAME = \'urn:perun:resource:attribute-def:core:name';
+
+# GATHER USERS
+my $users;  # $users->{$login}->{ATTR} = $attrValue;
+
+# AGGREGATE DATA
+#
+foreach my $resourceId ( $data->getResourceIds() ){
+	# Iterate through resources assigned to the facility.
+
+	my $resourceName = $data->getResourceAttributeValue(resource => $resourceId, attrName => $A_R_NAME);
```

Comment:
Is resource really represented in the external system by its name in the Perun? It should be an attribute on the resource IMO.